### PR TITLE
Consume ObjectiveGit through a binary Swift package

### PR DESCRIPTION
**Pull request by SimplyDanny**
This is a build test for the changes introduced in the [ObjectiveGit repository](https://github.com/mssun/objective-git/pull/2). ObjectiveGit is consumed as an XCFramework by a Swift package.

### CODE REVIEW

**Comment by SimplyDanny** [comment]
@sanderdekoning: You may check this branch/PR out. With the use of the pre-compiled ObjectiveGit, I expect that you can just follow the updated build steps in the README to build and run the app locally (on Intel and Apple Silicon). Some of the dependencies as well as the `export PKG_CONFIG_PATH=...` should not be required anymore. So the setup is a bit simpler now and requires less manual steps.

It would be great if you could check whether it works for you. 😉

**Comment by sanderdekoning** [comment]
Thats awesome @SimplyDanny, I'll be sure to test it and report back here

**Comment by mssun** [comment]
I have merged the PR in objective-git and made a release: https://github.com/mssun/objective-git/releases/download/0.16-passforios/ObjectiveGit.xcframework.zip

Can you update the PR accordingly?

**Comment by SimplyDanny** [comment]
> Can you update the PR accordingly?

The PR has been updated. The The Swift package now references your released version.

File: pass.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1407,27 +1414,7 @@
...
-			);
-			name = "Copy Carthage Frameworks";
-			outputFileListPaths = (
-			);
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "/usr/local/bin/carthage copy-frameworks\n";
+			shellScript = "#. \"${SRCROOT}/scripts/swiftformat.sh\"\n";
```
Comment (mssun) [inline pass.xcodeproj/project.pbxproj:1417]: The shell script can be enabled now.

File: pass.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1445,7 +1432,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = ". \"${SRCROOT}/scripts/swiftlint.sh\"\n";
+			shellScript = "#. \"${SRCROOT}/scripts/swiftlint.sh\"\n";
```
Comment (mssun) [inline pass.xcodeproj/project.pbxproj:1435]: The shell script can be enabled now.

File: pass.xcodeproj/project.pbxproj (hunk 1 above)
Comment (SimplyDanny) [reply pass.xcodeproj/project.pbxproj]: Done.

File: pass.xcodeproj/project.pbxproj (hunk 2 above)
Comment (SimplyDanny) [reply pass.xcodeproj/project.pbxproj]: Done.

**Pull request merged**